### PR TITLE
Fix reports with missed values

### PR DESCRIPTION
**Pull request by Abyss777**
Chart generating brakes if some attributes missed in the set https://www.traccar.org/forums/topic/graphic-report-issue/
as well as route grid (just warning, but...)

ExtJS correctly handle missed values, but they should be `undefined`, not `null`

### CODE REVIEW

**Comment by Abyss777** [comment]
By the way, they are rendered as `gap` by default http://docs.sencha.com/extjs/6.2.0/classic/Ext.chart.series.Line.html#cfg-nullStyle
Do not know, why it is called null

File: web/app/controller/Root.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ Ext.define('Traccar.controller.Root', {
         if (value !== undefined) {
             return Traccar.AttributeFormatter.getAttributeConverter(this.attributeKey)(value);
         }
-        return null;
+        return undefined;
```
Comment (tananaev) [inline web/app/controller/Root.js]: Undefined value shouldn't be used explicitly. I propose we remove return statement instead.

**Comment by Abyss777** [comment]
Oh, sure. Replaced commit.

**Comment by Abyss777** [comment]
lint is failed than
````
/home/local/RG/kas/repos/traccar/traccar-web/web/app/controller/Root.js
  50:25  error  Expected to return a value at the end of method 'calculateAttribute'  consistent-return
````

**Comment by tananaev** [comment]
I know the solution. Just return `value`.

**Comment by Abyss777** [comment]
Good. Thanks for the tip.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**